### PR TITLE
Psp 8099 - allow special instruction whitespace to display.

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

File: source/backend/api/Models/Configuration/MayanConfig.cs [hunk 1]
```diff
@@ -11,5 +11,9 @@ public class MayanConfig
         public string ConnectionPassword { get; set; }
 
         public int UploadRetries { get; set; }
+
+        public int ImageRetries { get; set; }
+
+        public int PreviewPages { get; set; }
```
Comment (asanchezr) [inline source/backend/api/Models/Configuration/MayanConfig.cs:17]: These changes are unrelated - please rebase this branch

Comment (devinleighsmith) [reply source/backend/api/Models/Configuration/MayanConfig.cs]: this has been corrected.

**Review by asanchezr — APPROVED** [review]
